components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -1746,22 +1746,24 @@ public static String compressResponse(String response) throws IOException {
     }
 
     public static String getNotificationEndpoint(){
-        String redirectURL = IdentityUtil.getProperty(IdentityConstants.ServerConfig
-                .NOTIFICATION_ENDPOINT);
-        if (StringUtils.isBlank(redirectURL)){
-            redirectURL = IdentityUtil.getServerURL(SAMLSSOConstants.NOTIFICATION_ENDPOINT, false, false);
+
+        try {
+            return resolveUrl(SAMLSSOConstants.NOTIFICATION_ENDPOINT, IdentityConstants.ServerConfig
```

Comment:
shouldn't the ServerConfig be called through IdentityUtil.getProperty() method?

Reply:
ahh good point :)